### ISSUE TITLE
Memcache constructor type does not fully express its valid uses

### ISSUE THREAD

**Issue by iamEAP** [jaredwray/keyv#277]
Got another bump that breaks on typescript compilation, this time on the memcache sub-package (https://github.com/backstage/backstage/pull/10705).

Specific error is:

```
147       store: new KeyvMemcache(this.connection),
                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  node_modules/keyv-memcache/src/index.d.ts:7:30
    7     constructor(uri: string, opts: KeyvMemcache.Options);
                                   ~~~~~~~~~~~~~~~~~~~~~~~~~~
    An argument for 'opts' was not provided.
```

**Comment by jaredwray** [comment]
@iamEAP - we will get a fix for this out today.

**Comment by jaredwray** [comment]
@iamEAP - just published an update to NPM (version 1.3.2). Can you validate this is fixed?
